openstack/provision.sh

```diff
@@ -5,10 +5,10 @@ SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
 source "$SCRIPT_DIR/common.sh"
 
 echo "Provisioning $EIRINI_STATION_USERNAME-eirini-station"
-scp -r ~/workspace/eirini-station/provision*.sh "$EIRINI_STATION_USERNAME@$STATION_IP:/tmp"
-ssh -A "$EIRINI_STATION_USERNAME@$STATION_IP" "sudo /tmp/provision.sh"
-ssh -A "$EIRINI_STATION_USERNAME@$STATION_IP" "/tmp/provision-user.sh"
+scp -i /Users/d062748/.ssh/ccee_privkey -r ~/git/eirini-station/provision*.sh "$VMUSER@$STATION_IP:/tmp" 
+ssh -i /Users/d062748/.ssh/ccee_privkey -A "$VMUSER@$STATION_IP" "sudo /tmp/provision.sh" 
+ssh -i /Users/d062748/.ssh/ccee_privkey -A "$VMUSER@$STATION_IP" "/tmp/provision-user.sh"
```

Comment:
We should not assume that the private key is stored locally in `.ssh`. Instead we can realy that is has been added using `ssh-add <key-file>`. In this case we can drop the `-i` flag completely.

Reply:
removed the -i flag